README.md

```diff
@@ -149,26 +147,21 @@ HiQ was originally developed to find Onnxruntime performance bottleneck in DNN i
 - [Latency](https://github.com/oracle-samples/hiq/blob/main/hiq/examples/onnxruntime/demo.ipynb)
 - [Intrusive](https://github.com/oracle-samples/hiq/blob/main/hiq/examples/onnxruntime/demo_intrusive.ipynb)
 
-
 ## Examples
 
 Please check [🔗 examples](https://github.com/oracle/hiq/blob/main/hiq/examples) for usage examples.
 
 ## Contributing
 
-
-HiQ welcomes contributions from the community. Before submitting a pull request, please [review our 🔗 contribution guide](https://github.com/oracle/hiq/blob/main/CONTRIBUTING.md).
-
-
+HiQ welcomes contributions from the community. Before submitting a pull request, please review our contribution guide](./CONTRIBUTING.md).
```

Comment:
Thanks @spavlusieva in README.md, we should use the full URL, otherwise, the document in readthedocs and pypi would have dead links. This is very tricky tip which I learned after some error and trail.